### PR TITLE
Certificate Page template Integration

### PR DESCRIPTION
**Pull request by asadiqbal08**
#### What are the relevant tickets?
#1109 #1095 #1103 

#### What's this PR do?
Designed the bootcamp certificate and integrated the CMS stuff in it.
Please look around the ticket acceptance criteria to verify changes.

### UPDATED SCREENSHOTS
Desktop:
image

Mobile:
image

Print:
image

#### Old Screenshots (Web View):

screencapture-bc-odl-local-8099-cms-pages-14-edit-preview-2020-12-24-17_21_01
screencapture-bc-odl-local-8099-cms-pages-14-edit-preview-2020-12-24-17_21_57
screencapture-bc-odl-local-8099-cms-pages-14-edit-preview-2020-12-24-17_22_15

#### Old Screenshot (Mobile View):
screencapture-bc-odl-local-8099-cms-pages-14-edit-preview-2020-12-24-17_23_41

#### Old Screenshot (Print Version):

Screen Shot 2020-12-24 at 5 24 25 PM

### CODE REVIEW

File: static/scss/certificates/certificate.scss [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+      height: 50px;
+      object-fit: contain;
+    }
+  }
+
+  .signature-text {
+    font-weight: bold;
+    font-size: 16px;
+    line-height: 22px;
+    color: #333;
```
Comment (HamzaIbnFarooq) [inline static/scss/certificates/certificate.scss:295]: We should move Hex values to `_variables.scss`

**Review by HamzaIbnFarooq — CHANGES_REQUESTED** [review]
I want some understanding regarding the following point:
1) Do we want any header and footer to be shown on certificate page (on platform/not printed)? (As there isn't any right now)
2) As xPro has a boundary of the certificate, do we need to create a boundary here as well?
3) I am unable to print certificate with the black boundary, similar to your screenshot, why is it so?
4) The help button is also getting printed, don't we need to commit it as well?

File: cms/models_test.py [hunk 2]
```diff
@@ -303,11 +303,11 @@ def test_certificate_for_bootcamp_run_page():
         assert signatory.value.signature_image.title == "Image"
 
 
-def test_certificate_index_page(rf):
+def test_certificate_index_page(rf, settings):
     """
     test for certificate index page
     """
-
+    settings.FEATURES["ENABLE_CERTIFICATE_USER_VIEW"] = True
```
Comment (HamzaIbnFarooq) [inline cms/models_test.py:310]: can we use override_settings here?

File: cms/models_test.py [hunk 3]
```diff
@@ -325,9 +325,17 @@ def test_certificate_index_page(rf):
...
     )
+
     with pytest.raises(Http404):
         certifcate_index_page.bootcamp_certificate(
             request, "00000000-0000-0000-0000-000000000000"
         )
+
+    # Revoke the certificate and check index page returns 404
+    certificate.is_revoked = True
+    certificate.save()
```
Comment (HamzaIbnFarooq) [inline cms/models_test.py:336]: Please use `certificate.revoke()` instead.

Comment (arslanashraf7) [reply cms/models_test.py]: Updated

File: cms/models_test.py (hunk 2 above)
Comment (arslanashraf7) [reply cms/models_test.py]: Updated

**Review by HamzaIbnFarooq — APPROVED** [review]
Final Review:
👍

**Pull request merged**